configure.ac

```diff
@@ -7,8 +7,8 @@
 AC_PREREQ([2.63])
 
 define([scm_r], esyscmd([sh -c "git rev-parse --short=7 HEAD"]))
-define([ucx_ver_major], 0)
-define([ucx_ver_minor], 1)
+define([ucx_ver_major], 1)
```

Comment:
IMO - too small change for separate PR.
now version is in par w/ github milestones.